examples/column/bb_fct_advection.jl

```diff
@@ -87,54 +89,70 @@ domain = Domains.IntervalDomain(
     Geometry.ZPoint{FT}(π);
     boundary_names = (:bottom, :top),
 )
-mesh = Meshes.IntervalMesh(domain, nelems = n)
-cs = Spaces.CenterFiniteDifferenceSpace(mesh)
-fs = Spaces.FaceFiniteDifferenceSpace(cs)
-zc = Fields.coordinate_field(cs)
-
-# Unitary, constant advective velocity
-w = Geometry.WVector.(speed .* ones(FT, fs))
-
-# Define a pulse wave or square wave
-pulse(z, t, z₀, zₕ, z₁) = abs(z.z - speed * t) ≤ zₕ ? z₁ : z₀
-
-# Initial condition
-y0 = pulse.(zc, 0.0, z₀, zₕ, z₁)
-
-# ClimaTimeSteppers need a FieldVector
-y0 = Fields.FieldVector(y = y0)
-
-# Set up parameters needed for time-stepping
-Δt = 0.0001
-dydt = copy(y0)
-y_td = similar(dydt.y)
 
-parameters = (; w, y_td)
-# Call the RHS function
-f!(dydt, y0, parameters, 0.0, Δt, 1)
-t₁ = 100Δt
-prob = ODEProblem(IncrementingODEFunction(f!), copy(y0), (t₀, t₁), parameters)
-sol = solve(
-    prob,
-    SSPRK33ShuOsher(),
-    dt = Δt,
-    saveat = Δt,
-    progress = true,
-    adaptive = false,
-    progress_message = (dt, u, p, t) -> t,
-)
-computed_result = sol.u[end].y
-analytical_result = pulse.(zc, t₁, z₀, zₕ, z₁)
-err = norm(computed_result .- analytical_result)
-initial_mass = sum(sol.u[1].y)
-mass = sum(sol.u[end].y)
-rel_mass_err = norm((mass - initial_mass) / initial_mass)
-
-@test err ≤ 0.0185
-@test rel_mass_err ≤ 8eps()
-
-plot(sol.u[end].y)
-Plots.png(
-    Plots.plot!(analytical_result, title = "Boris and Book FCT"),
-    joinpath(path, "exact_and_computed_advected_square_wave_BBFCT.png"),
-)
+stretch_fns = (Meshes.Uniform(), Meshes.ExponentialStretching(FT(7.0)))
+err = zeros(FT, length(stretch_fns))
+initial_mass = zeros(FT, length(stretch_fns))
+mass = zeros(FT, length(stretch_fns))
+rel_mass_err = zeros(FT, length(stretch_fns))
+plot_string = ["uniform", "stretched"]
+
+for (i, stretch_fn) in enumerate(stretch_fns)
+
+    mesh = Meshes.IntervalMesh(domain, stretch_fn; nelems = n)
+    cs = Spaces.CenterFiniteDifferenceSpace(mesh)
+    fs = Spaces.FaceFiniteDifferenceSpace(cs)
+    zc = Fields.coordinate_field(cs)
+
+    # Unitary, constant advective velocity
+    w = Geometry.WVector.(speed .* ones(FT, fs))
+
+    # Define a pulse wave or square wave
+    pulse(z, t, z₀, zₕ, z₁) = abs(z.z - speed * t) ≤ zₕ ? z₁ : z₀
```

Comment:
Not super important, but `pulse ` could be moved out of the for loop since it's not making a closure over any variables in the for loop

Reply:
Right. I had actually thought about and meant to do it, but forgot in that moment. Thanks for the reminder!

Reply:
Done